ps_buybuttonlite.php

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+* 2007-2018 PrestaShop
+*
+* NOTICE OF LICENSE
+*
+* This source file is subject to the Academic Free License (AFL 3.0)
+* that is bundled with this package in the file LICENSE.txt.
+* It is also available through the world-wide-web at this URL:
+* http://opensource.org/licenses/afl-3.0.php
+* If you did not receive a copy of the license and are unable to
+* obtain it through the world-wide-web, please send an email
+* to license@prestashop.com so we can send you a copy immediately.
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+*  @author    PrestaShop SA <contact@prestashop.com>
+*  @copyright 2007-2018 PrestaShop SA
+*  @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+*  International Registered Trademark & Property of PrestaShop SA
+*/
+
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
+
+class Ps_BuybuttonLite extends Module
+{
+    public function __construct()
+    {
+        $this->name = 'ps_buybuttonlite';
+        $this->tab = 'administration';
+        $this->version = '1.0.0';
+        $this->author = 'PrestaShop';
+        $this->need_instance = 0;
+
+        $this->module_key = 'bab535c26e031e9d224c0d128e063283';
+
+        $this->bootstrap = true;
+
+        parent::__construct();
+
+        $this->controllerAdmin = 'AdminAjaxPs_buybuttonlite';
+
+        $this->displayName = $this->trans('Buy button lite', array(), 'Modules.Buybuttonlite.Admin');
+        $this->description = $this->trans('Create a product checkout redirect link to post on a blog or any social media', array(), 'Modules.Buybuttonlite.Admin');
+        $this->ps_version = (bool)version_compare(_PS_VERSION_, '1.7', '>=');
+
+        // Settings paths
+        $this->css_path = $this->_path.'views/css/';
+
+        // Confirm uninstall
+        $this->confirmUninstall = $this->trans('Are you sure you want to uninstall this module?', array(), 'Modules.Buybuttonlite.Admin');
```

Comment:
Already localized in _Modules.Legalcompliance.Admin_. No need to duplicate, you can use this first domain. ;-)

Reply:
Changed, thanks!